float32 e_x # controller error for X axis
float32 e_y # controller error for Y axis
float32 e_z # controller error for Z axis
float32 v_x_cmd # commanded vel for X axis
float32 v_y_cmd # commanded vel for Y axis
float32 v_z_cmd # commanded vel for Z axis
float32 pos_x # position in X axis
float32 pos_y # position in Y axis
float32 pos_z # position in Z axis